gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -219,4 +219,20 @@ MM_ConfigurationGenerational::defaultMemorySpaceAllocated(MM_GCExtensionsBase *e
 	extensions->setGuaranteedNurseryRange(nurseryStart, (void*)UDATA_MAX);
 }
 
+uintptr_t
+MM_ConfigurationGenerational::calculateDefaultRegionSize(MM_EnvironmentBase *env)
+{
+	uintptr_t regionSize = STANDARD_REGION_SIZE_BYTES;
+
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)

```

Comment:
Well, in some places that API (ext->isConcurrentScavengerEnabled()) cannot be used... if symbols are defined for CS only, like here concurrentScavengerPageSectionSize.